models/modelling/commissioning/prescribing/int_person_prescribing_cost_monthly.sql

```
@@ -20,6 +21,7 @@ before relying on recent months.
 select
     sk_patient_id
     , date_trunc('month', processing_period_date)   as activity_month
+    , 'actual'                                      as cost_basis
```

SQLFluff (4.2.2)

[error] Found leading comma ','. Expected only trailing near line breaks.

(LT04)